takwimu/urls.py

```diff
@@ -23,7 +23,10 @@
     url(r'^about/support-services',
         SupportServicesIndexView.as_view(),
         name='about_supportservices'),
-    url(r'^about/?$', AboutUsView.as_view(), name='about'),
+
+    # Use AboutPage model
+    # url(r'^about/?$', AboutUsView.as_view(), name='about'),
```

Comment:
I think it worth removing this or find a better way to document this.

Reply:
Removed.